src/app/oauth2/middleware.js

```diff
@@ -17,8 +17,10 @@ module.exports = {
 
   parseSharedAttributesJWT: async (req, res, next) => {
     const requestJWT = req.query.request;
+    const headers = {'client_id': req.session?.authParams?.client_id }
```

Comment:
Could you just add a space after the open curly bracket? And then also it just needs a semicolon at the end of this line.